react/ActionMenu/Readme.md

```diff
@@ -2,224 +2,105 @@ Use an ActionMenu to show a list of actions. ActionMenus automatically switch th
 
 ### Classic
```

Comment:
Now that there is only one category, then having `Classic` as section name seems not to be relevant

If I read `Classic` my first reaction is "hey are there other modes available?"